jcl/src/java.base/share/classes/java/lang/invoke/MemberName.java

```diff
@@ -29,6 +29,12 @@
  */
 
 final class MemberName {
+	/*[IF Sidecar18-SE-OpenJ9]*/
+	static final class Factory {
```

Comment:
This wasn't previously needed: perhaps the condition ought to be `! Sidecar19-SE-OpenJ9`?

Reply:
A positive condition is better than a negative one, as the negative condition could end up including the code in unexpected places.